mesa/visualization/templates/js/NetworkModule_d3.js

```diff
@@ -80,6 +109,13 @@ var NetworkModule = function (svg_width, svg_height) {
       })
       .attr("stroke", function (d) {
         return d.color;
+      })
+      .attr("marker-end", function (d) {
+        return d.directed
+          ? "url(#end" +
+              (d.marker_size !== undefined ? d.marker_size : "") +
```

Comment:
You can do this instead
`(d.marker_size ?? "")`

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator